### DISCUSSION TITLE
WORKFLOW_DISPATCH.md

### DISCUSSION THREAD

**Discussion by samuelcastro** [UnlyEd/next-right-now#256] [General] [answered]
Why was the `.github/WORKFLOW_DISPATCH.md` file removed?

The documentation is still mentioning it: 
https://github.com/UnlyEd/next-right-now/blob/v2-mst-aptd-gcms-lcz-sty/.github/workflows/deploy-vercel-production.yml#L42

Was it a mistake?

**Comment by Vadorequest** [comment, marked as answer]
I forgot to update the link, thanks for the feedback!

Doc has been moved to https://unlyed.github.io/next-right-now/guides/ci-cd/gha-deploy-vercel#triggering-the-action-remotely-using-workflow_dispatch

**Reply by samuelcastro** [reply]
Ok great.
